tot/chrome.chromium.go

```diff
@@ -173,6 +173,19 @@ func (chrome *Chrome) GetTab(tabID string) (Tabber, error) {
 	return tab, err
 }
 
+/*
+RemoveTab implements Chromium.
+*/
+func (chrome *Chrome) RemoveTab(tab *Tab) {
+	tabs := make([]*Tab, 0)
+	for _, t := range chrome.Tabs() {
+		if t != tab {
+			tabs = append(tabs, t)
+		}
+	}
+	chrome.tabs = tabs
+}
```

Comment:
Rather than iterating through the tabs and creating a new list, I think it's more efficient and cleaner to simply remove the specific tab from the current list once it's found, something like 

```go
for k, t := range chrome.tabs {
	if t == tab {
		chrome.tabs = append(chrome.tabs[:k], chrome.tabs[k+1:])
		break
	}
}
```